wear/src/main/AndroidManifest.xml

```diff
@@ -10,15 +10,46 @@
     android:label="@string/localised_app_name"
     android:supportsRtl="true"
     android:theme="@android:style/Theme.DeviceDefault">
+
     <activity
-      android:name=".MainActivity"
+      android:name=".ContactEventsActivity"
       android:label="@string/localised_app_name">
       <intent-filter>
         <action android:name="android.intent.action.MAIN" />
-
         <category android:name="android.intent.category.LAUNCHER" />
       </intent-filter>
     </activity>
+
+    <service
+      android:name=".ContactEventsProviderService"
+      android:icon="@drawable/ic_complication_contact_event"
+      android:label="@string/complications_provider_contact_events">
+
+      <intent-filter>
+        <action android:name="android.support.wearable.complications.ACTION_COMPLICATION_UPDATE_REQUEST" />
+      </intent-filter>
+
+      <meta-data
+        android:name="android.support.wearable.complications.SUPPORTED_TYPES"
+        android:value="SHORT_TEXT,LONG_TEXT" />
```

Comment:
This is where we specify to the system the complications types we can provide data for.
Again, this PR is a first implementation, it would be good to have more supported types in the future.